migrations/014-create-commentreaction.js

```diff
@@ -0,0 +1,44 @@
+module.exports = {
+  up: (queryInterface, Sequelize) => queryInterface
+    .createTable('Commentreactions', {
```

Comment:
Make `Commentreactions` camel cased. There are two words there

Reply:
ok... tot I had to make an exception since its a table name
renaming rightaway